src/commands/play.rs

```diff
@@ -102,6 +118,42 @@ pub async fn execute_play(
     Ok(())
 }
 
+async fn calculate_time_until_play(queue: &[TrackHandle], flag: &PlayFlag) -> Option<Duration> {
+    if !queue.is_empty() {
+        let top_track = queue.first().expect("Could not fetch playing song");
+
+        let top_track_position = top_track
+            .get_info()
+            .await
+            .expect("Could not get playing track info")
+            .position;
+
+        let top_track_duration = top_track
+            .metadata()
+            .duration
+            .expect("Could not fetch duration of top track");
+
+        let mut estimated_time = match flag {
+            PlayFlag::DEFAULT => queue[1..queue.len() - 1]
```

Comment:
Can just do `queue[1..]`

Reply:
Nope, I don't want the last track, as it is the last inserted track.

Reply:
:donkey: me